pkg/ipamd/ipamd.go

```diff
@@ -655,7 +646,15 @@ func (c *IPAMContext) StartNodeIPPoolManager() {
 }
 
 func (c *IPAMContext) updateIPPoolIfRequired(ctx context.Context) {
-	c.askForTrunkENIIfNeeded(ctx)
+	if c.enablePodENI {
+		c.askForTrunkENIIfNeeded(ctx)
```

Comment:
For the same reason, we let CNI tries to send another event in 30s instead of retry or handle error at this place since the error should come from `SendNodeEvent()`.

Reply:
So, we're banking on the reconciler triggering `updateIPPoolIfRequired` every 30s?

Reply:
yes, Apurup. We can discuss on this offline